docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -109,7 +108,7 @@ The time interval that the maximum is computed over is set by the `step` paramet
 The `avg_over_time()` function lets you aggregate numerical values by computing the maximum value of them, such as the all important span duration.
 The time interval that the maximum is computer over is set by the `step` parameter.
 
-For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics).
```

Comment:
These changes can be discarded, they make no difference to the final URL.

Reply:
Done.